src/test/ui/lint/cli-unknown-force-warn.stderr

```diff
@@ -0,0 +1,15 @@
+error[E0602]: unknown lint: `foo_qux`
+   |
+   = note: requested on the command line with `--force-warns foo_qux`
+
+error[E0602]: unknown lint: `foo_qux`
```

Comment:
Any idea why this error gets emitted multiple times?

Reply:
This happens for [some other cases as well](https://github.com/rust-lang/rust/blob/40c1623b16fb28f7625cab13570752d04513815c/src/test/ui/lint/lint-unknown-lint-cmdline.stderr). I think it's an artifact of the way the test is run, though I'm not totally sure?